src/components/Wowerlay.tsx

```diff
@@ -111,6 +111,26 @@ export const Wowerlay = defineComponent({
     };
   },
   render() {
+    let willBeRendered: false | JSX.Element = false;
```

Comment:
Why not `undefined` or `null`? Isn't `false` the wrong choice to express the absence of an element?

Reply:
Because of `visibility` condition, this variable can only get JSX.Element or false
```ts
    const Renderer = this.isVisible && (
      <WowerlayRenderer {...this.$props} {...this.$attrs} onClick={this.handleWowerlayClick}>
        {this.$slots.default?.()}
      </WowerlayRenderer>
    );
```

Reply:
https://github.com/teamseodo/wowerlay/pull/35/files#diff-44118b4b83012845c16026a031935f131e3ef49217807632f631a2d8a4e8f6eaR123

Reply:
```jsx
  const Renderer = !this.isVisible ? null : (
      <WowerlayRenderer {...this.$props} {...this.$attrs} onClick={this.handleWowerlayClick}>
        {this.$slots.default?.()}
      </WowerlayRenderer>
    );
```

Reply:
https://github.com/teamseodo/wowerlay/pull/35/commits/8d3316cd97747865c0a86e14c1bdfc17d4194b4e